SocialCareCaseViewerApi/V1/Gateways/IDatabaseGateway.cs

```diff
@@ -22,5 +23,6 @@ public interface IDatabaseGateway
         List<Team> GetTeams(string context);
         UpdateAllocationResponse UpdateAllocation(UpdateAllocationRequest request);
         CreateWarningNoteResponse CreateWarningNote(CreateWarningNoteRequest request);
+        List<WarningNote> GetWarningNotes(GetWarningNoteRequest request);
```

Comment:
As per comment on DatabaseGateway this would become IEnumerable<WarningNote>

Reply:
Great suggestion, I have updated the code